bin/test-code-style

```diff
@@ -5,31 +5,15 @@
 # shellcheck source=bin/scriptLib
 . "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/scriptLib"
 
-## The following are three separate `sbt` invocations because in some versions of sbt (0.13)
-## trying to run all tasks in a single session caused "GC overhead limit exceeded" (both in
-## Travis and locally).
-## See https://github.com/lagom/lagom/pull/1482
-
-runSbt  +headerCheck \
-        +test:headerCheck \
-        multi-jvm:headerCheck
-
 declare -a EXTRA_TASKS
 if [ "$TRAVIS_EVENT_TYPE" != "cron" ]; then
     EXTRA_TASKS+=("versionSyncCheck")
 fi
 
-runSbt  "${EXTRA_TASKS[@]}" \
-        checkAkkaModuleVersions \
-        +mimaReportBinaryIssues
-
-printMessage "VALIDATE SCALA CODE FORMATTING"
-runSbt scalafmtAll scalafmtSbt
-git diff --exit-code || (
-  echo "[error] ERROR: scalafmt check failed, see differences above."
-  echo "[error] To fix, format your sources using 'sbt scalafmtAll scalafmtSbt' before submitting a pull request."
-  false
-)
+runSbt  +headerCheck +test:headerCheck multi-jvm:headerCheck \
+        scalafmtCheckAll scalafmtSbtCheck \
+        +mimaReportBinaryIssues \
+        checkAkkaModuleVersions "${EXTRA_TASKS[@]}"
```

Comment:
(Minor) add `javafmt test:javafmt multi-jvm:javafmt` and also `whitesourceUpdate` via `EXTRA_TASKS`?  (You can set "noisy" with the `info` command, which sets the log level back from warn to info.)

Reply:
The problem is that all the other tasks are _checking_ something, but `javafmt` does not _fail_ if the code is not formatted accordingly. So better to keep it separated.

Reply:
Yeah, I'm not saying you can get rid of the follow-on `git diff --exit-code`.  But nothing wrong with keeping it separate either.

BTW `javafmtCheckAll` is on Santa's list: https://github.com/sbt/sbt-java-formatter/pull/53